### PR TITLE
Dropdown from harmony

### LINKED ISSUE #774: Dropdown from harmony
Dropdown, DropdownTrigger, DropdownPanel, ListView, ListViewItem, MenuItem, Combobox from harmony

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #774 

## PR includes

-   [x] Refactor

### CODE REVIEW

File: src/components/FormGradeDropdown/FormGradeDropdown.tsx [hunk 1]
```diff
@@ -1,48 +1,65 @@
...
-    },
-);
+            <DropdownPanel placement="bottom-start">
+                <ListView>
+                    {gradeOptions.map((item, index) => (
+                        <ListViewItem
+                            key={index}
+                            value={item}
+                            renderItem={({ active, hovered, ...props }) => (
+                                <MenuItem
```
Comment (asabotovich) [inline src/components/FormGradeDropdown/FormGradeDropdown.tsx:48]: why you don't use active and hovered?

File: src/components/TimePicker/TimePicker.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}: TimePickerProps): JSX.Element => {
+    const options = useMemo(() => getOptions(), []);
+    const [value, setValue] = useState<string>(getStringTime(initialValue.getHours(), initialValue.getMinutes()));
+    const handleChange = (eventValue: string): void => {
+        setValue(eventValue as string);
+        onChange(splitDateValue(eventValue as string));
+    };
+
+    return (
+        <div>
```
Comment (asabotovich) [inline src/components/TimePicker/TimePicker.tsx:30]: why do you need this div?

File: src/components/CrewUserSelector/CrewUserSelector.tsx [hunk 3]
```diff
@@ -29,65 +35,69 @@ export const CrewUserSelector = ({ onSelect, placeholder }: CrewUserSelectorProp
...
-        reset();
-    };
+    const onUserSelect = useCallback(
+        async (crewUser: CrewUserShort) => {
+            const user = await getUserByCrewUser.mutateAsync(crewUser);
+            onSelect(user);
+            reset();
+        },
+        [onSelect, reset],
+    );
```
Comment (asabotovich) [inline src/components/CrewUserSelector/CrewUserSelector.tsx:49]: warning

File: src/components/AddOrUpdateCandidate/AddOrUpdateCandidate.module.css [hunk 4]
```diff
@@ -4,3 +4,10 @@
 
 }
 
+.SelectVendor {
+    padding-left: var(--gap-m);
+    display: flex;
+    background-color: var(--gray3);
```
Comment (asabotovich) [inline src/components/AddOrUpdateCandidate/AddOrUpdateCandidate.module.css:10]: legacy-colors

**Review by asabotovich — CHANGES_REQUESTED** [review]

File: src/components/TimePicker/TimePicker.tsx (hunk 2 above)
Comment (nyet-ty) [reply src/components/TimePicker/TimePicker.tsx:30]: to have text above select

File: src/components/FormGradeDropdown/FormGradeDropdown.tsx (hunk 1 above)
Comment (nyet-ty) [reply src/components/FormGradeDropdown/FormGradeDropdown.tsx:62]: forgot about it:/

**Review by asabotovich — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**